doc/run/k8s/install-sqlflow.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
Is YAML section to add a namespace? How about we add a comment on why we need to create a namespace?

Reply:
A namespace provides a scope for Kubernetes resource e.g. Pod, Service, but thanks for your reminding, I think we don't need a namespace `sqlflow` Here, SQLFlow can run on the default namespace.